dll/cpl/usrmgr/groupprops.c

```diff
@@ -9,6 +9,10 @@
 
 #include "usrmgr.h"
 
+#include "wine/debug.h"
+#include "wine/unicode.h"
+WINE_DEFAULT_DEBUG_CHANNEL(usrmgr);
+
```

Comment:
You can put them in `usrmgr.h` once and for all.

Reply:
Right